packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -149,9 +149,20 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
   Animation<double>? get secondaryAnimation => _secondaryAnimation;
   final ProxyAnimation _secondaryAnimation = ProxyAnimation(kAlwaysDismissedAnimation);
 
+  /// Whether to takeover the [controller] created by [createAnimationController].
+  ///
+  /// If true, this route is responsible for the disposal of the controller.
```

Comment:
If true, this route will call [AnimationController.dispose] when the controller is no longer needed.

I think that's all you need to say here.